docs/src/architecture/YubicoAuthentication.md

```diff
@@ -0,0 +1,34 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Yubico Passwordless Authentication
+
+This section describes Yubico Passwordless Authentication and how to create U2F keys.
```

Comment:
Let's make this a bit more generic - "This section describes Ghaf reference implementation for passwordless authentication for fast-security online (FIDO). The reference implementation has been created with Yubico Yubikeys.
The implementation is modular and configurable - thus enabling also other implementations."

Please also add references to FIDO tech agnostic specs in the "The reader is expected to know the fundamentals ..." part.

Reply:
Updated as per review comment.